WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -225,6 +225,7 @@ def post_results
       end
     end
     comp.update!(results_posted_at: Time.now)
+    comp.competitor_users.each { |user| user.notify_of_results_presence(comp) }
```

Comment:
We use the word "posted" everywhere else, how do you feel about `notify_of_results_posted` instead of `notify_of_results_presence` for consistency?